css/style.css

```diff
@@ -0,0 +1,81 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  font-family: Helvetica, Helvetica-Neue, Arial;
+  text-align: center;
+}
+
```

Comment:
I don't recommend having newlines between CSS definitions. It doesn't matter, technically, but it's a style thing that the interwebs have agreed upon